Clicker/ViewControllers/PollBuilderViewController.swift

```diff
@@ -315,8 +316,31 @@ class PollBuilderViewController: UIViewController, QuestionDelegate, PollBuilder
     }
     
     @objc func toggleQuestionType() {
-        dropDown.isHidden = false
-        
+        let width = ModalSize.full
+        let height = ModalSize.custom(size: editQuestionTypeModalHeight)
+        let originY = 0
+        let center = ModalCenterPosition.customOrigin(origin: CGPoint(x: 0, y: originY))
+        let customType = PresentationType.custom(width: width, height: height, center: center)
+        let presenter = Presentr(presentationType: customType)
+        presenter.backgroundOpacity = 0.6
+        presenter.transitionType = .coverVerticalFromTop
+        let editQuestionTypeVC = EditQuestionTypeViewController(selectedQuestionType: questionType)
+        editQuestionTypeVC.delegate = self
+        customPresentViewController(presenter, viewController: editQuestionTypeVC, animated: true, completion: nil)
+    }
+    
+    func didPick(questionType: QuestionType) {
+        self.questionType = questionType
+        updateQuestionTypeButton()
+        switch questionType {
+        case .multipleChoice:
+            mcPollBuilder.isHidden = false
+            frPollBuilder.isHidden = true
+            break
+        case .freeResponse:
+            mcPollBuilder.isHidden = true
+            frPollBuilder.isHidden = false
+        }
```

Comment:
You can reduce this down to:
`mcPollBuilder.isHidden = questionType == .freeResponse`
`frPollBuilder.isHidden = questionType == . multipleChoice`